csrc/host_ir/executor.h

```diff
@@ -108,6 +108,10 @@ class HostIrEvaluator final : public OptOutDispatch {
     return streams_;
   }
 
+  HostIrContainer* container() {
+    return container_.get();
+  }
+
```

Comment:
```suggestion
```

Reply:
I don't think you need this any more for this PR. But if another PR needs it I'm happy to review that. Same for hirEvaluator.